### PR TITLE
Create Dockerfile

### PR DESCRIPTION
**Pull request by kjellberg**
https://github.com/mrsked/mrsk/issues/62#issuecomment-1451814067

Usage:

```
docker build mrsk .
docker run -it -v "$PWD:/workdir" -v "/var/run/docker.sock:/var/run/docker.sock:rw" mrsk help
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+# Copy the rest of your application code into the container
+COPY . .
+
+# Install the gem locally from the project folder
+RUN gem build mrsk.gemspec && \
+    gem install ./mrsk-*.gem --no-document
+
+# Set the entrypoint to run the installed binary
+ENTRYPOINT ["mrsk"]
```
Comment (99linesofcode) [inline Dockerfile:32]: Adding a newline to the end of a file is generally considered good practice.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Use the official Ruby 3.2.0 Alpine image as the base image
+FROM ruby:3.2.0-alpine
+
+# Install system dependencies
+RUN apk add --no-cache build-base
```
[the remark this answers is not in the document]
Comment (kjellberg) [reply Dockerfile:5]: Thanks, I'll make an update soon

**Comment by kjellberg** [comment]
> I think it would make sense to use this MR to also introduce the Github Actions you mentioned in #62. Let's see what comes out of the discussions there.
> 
> I could lend a hand with that but I'm afraid I'm not going to be of much help when it comes to writing the Ruby code. Have to brush up on that again.

Thanks for your input! I was thinking of adding the Github Action via this PR as well, however I think that's something DHH has to do since we probably need env secrets in the repo etc?

**Comment by dhh** [comment]
Great stuff! I don't think we need to combine commands more than is done so far. These are distinct layers.

**Comment by dhh** [comment]
Great stuff! I don't think we need to combine commands more than is done so far. These are distinct layers.

**Pull request merged**